### PR TITLE
Pulling in items from PR11 to fix merge conflicts

### PR DESCRIPTION
**Pull request by aultt**
Merging in changes from PR11

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/xfailovercluster/22)

### CODE REVIEW

**Comment by aultt** [comment]
@mbreakey3 Change requested from @TravisEz13 has now been included and merge conflicts resolved can someone take a look and merge if no issues?  Thanks.

**Comment by TravisEz13** [comment]
The original commits should have retained the original​ author, but this is not blocking...
:lgtm: 
@mbreakey3   There is a new schema, can you review?

---

Reviewed 3 of 5 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xfailovercluster/22#-:-KO_tKHE4ZRuAp-Zb-62:b-q6ande)*

**Comment by MS-MatthewWalker** [comment]
Those options for quorum aren't correct for 2012R2 - The options are DiskWitness, FileShareWitness, NoWitness and DiskOnly the options listed in this code are correct for 2008R2 and 2012 but not later versions. Also there is no cloudwitness in the 2016 section.

**Comment by aultt** [comment]
This just a push from the original author.  Everything looks good for 2012R2.  Win2016 support will be added once it has released.

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xfailovercluster/22#-:-KOjHu2JFjNX95mx1SZd:bzavfnz)*

File: Tests/MSFT_xClusterQuorum.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+
+                $Result = Test-TargetResource @TestParameter
+
+                $Result | Should Be $true
+            }
+        }
+    }
+}
+
+Describe 'xClusterQuorum (NodeMajority / WS2016Prev)' {
```
Comment (Indhukrishna) [inline Tests/MSFT_xClusterQuorum.Tests.ps1]: What does WS2016Prev mean here?

**Comment by aultt** [comment]
*[Tests/MSFT_xClusterQuorum.Tests.ps1, line 142 \[r2\]](https://reviewable.io:443/reviews/powershell/xfailovercluster/22#-KOLtA2RlhcQ0buNjDd3-r2-142:-KOpbSVYhzqH-INuf4ei:bsyb6ec) ([raw file](https://github.com/powershell/xfailovercluster/blob/56064db948ce08dc1b7703037232127621402454/Tests/MSFT_xClusterQuorum.Tests.ps1#L142)):*
<details><summary><i>Previously, Indhukrishna (Indhu Sivaramakrishnan) wrote…</i></summary>
> What does WS2016Prev mean here? 
</details>
Windows Server 2016 Preview

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xfailovercluster/22)*

**Pull request merged**